File: Practice/The Best Internet Browser/csharp/BestInternetBrowser.cs
/*
* Problem Link - https://www.hackerearth.com/problem/algorithm/the-best-internet-browser-3/
*/


using System; 

class BestInternetBrowser 
{
	static void Main(string[] args)
        {
            var RawInput = System.Console.ReadLine().Trim();
            var testCases = Int32.Parse(RawInput);
            
            if (testCases >= 1 && testCases <= 100)
            {
                while (testCases-- > 0)
                {
                    var mystring = System.Console.ReadLine().Trim();
                    int originalLength = mystring.Length;
                    if (originalLength >= 1 && originalLength <= 200)
                    {
                        mystring = mystring.Replace("http://", "");
                        mystring = mystring.Replace("www.", "");
                        mystring = mystring.Replace(".com", "");
                        mystring = mystring.Replace("a", "");
                        mystring = mystring.Replace("e", "");
                        mystring = mystring.Replace("i", "");
                        mystring = mystring.Replace("o", "");
                        mystring = mystring.Replace("u", "");
                    }
                    Console.WriteLine((mystring.Length + 4) + "/" + originalLength);
                }
            }
        }
}

File: Practice/The Best Internet Browser/java/BestInternetBrowser.java
/*
* Problem Link - https://www.hackerearth.com/problem/algorithm/the-best-internet-browser-3/
*/

import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.PrintWriter;
 
class BestInternetBrowser {
 
	public static void main(String[] args) throws Exception {
		
		BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
		PrintWriter writer = new PrintWriter(System.out);
		
		int testCases = Integer.parseInt(reader.readLine());
		if (testCases >= 1 && testCases <= 100) {
			while (testCases-- > 0) {
				String mystring = reader.readLine();
				int originalLength = mystring.length();
				if (originalLength >= 1 && originalLength <= 200) {
					mystring = mystring.replace("http://", "");
					mystring = mystring.replace("www.", "");
					mystring = mystring.replace(".com", "");
					mystring = mystring.replace("a", "");
					mystring = mystring.replace("e", "");
					mystring = mystring.replace("i", "");
					mystring = mystring.replace("o", "");
					mystring = mystring.replace("u", "");
				}
				writer.write((mystring.length() + 4) + "/" + originalLength + "\n");
			}
		}
		reader.close();
		writer.close();
	}
}